homeassistant/components/qbittorrent/helpers.py

```diff
@@ -10,3 +13,48 @@ def setup_client(url: str, username: str, password: str, verify_ssl: bool) -> Cl
     # Get an arbitrary attribute to test if connection succeeds
     client.get_alternative_speed_status()
     return client
+
+
+def seconds_to_hhmmss(seconds) -> str:
```

Comment:
The parameter is missing a type annotation. Please type the whole signature when adding type annotations.